app/pandas.ipynb

```diff
@@ -208,18 +177,12 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 17,
-   "outputs": [],
-   "source": [
-    "# Write your code here"
-   ],
+   "source": "df[\"Average age\"] = df[\"Age\"].mean()",
```

Comment:
Adding 'Average age' as a column in the DataFrame might not be meaningful, as it will be the same for all rows. Consider calculating and displaying it separately instead.